### PR TITLE
CustomGlobalExceptionHandler, EmailValidator, PasswordValidator classes implemented

### PR DESCRIPTION
**Pull request by semyonich**

### CODE REVIEW

**Review by AndreiYazadzhi — APPROVED** [review]

File: src/main/java/com/cinema/util/EmailValidator.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import com.cinema.lib.EmailValidation;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements
+        ConstraintValidator<EmailValidation, String> {
+    @Override
+    public boolean isValid(String contactField,
+                           ConstraintValidatorContext cxt) {
+        return contactField != null && contactField.matches("\\w+@[a-z]+\\.[a-z]+")
```
Comment (boroda4436) [inline src/main/java/com/cinema/util/EmailValidator.java]: let's make the regex constant

File: src/main/java/com/cinema/util/PasswordValidator.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+public class PasswordValidator implements ConstraintValidator<PasswordValidation, Object> {
+    private String field;
+    private String fieldMatch;
+
+    public void initialize(PasswordValidation constraintAnnotation) {
+        this.field = constraintAnnotation.field();
+        this.fieldMatch = constraintAnnotation.fieldMatch();
+    }
+
+    public boolean isValid(Object value,
```
Comment (boroda4436) [inline src/main/java/com/cinema/util/PasswordValidator.java]: you can inline it if checkstyle allows
```suggestion
    public boolean isValid(Object value, ConstraintValidatorContext context) {
```

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: src/main/java/com/cinema/util/EmailValidator.java (hunk 1 above)
Comment (semyonich) [reply src/main/java/com/cinema/util/EmailValidator.java]: done

File: src/main/java/com/cinema/util/PasswordValidator.java (hunk 2 above)
Comment (semyonich) [reply src/main/java/com/cinema/util/PasswordValidator.java]: done

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**